src/drawer.multicanvas.js

```diff
@@ -301,10 +301,7 @@ export default class MultiCanvas extends Drawer {
                         peaks[Math.floor(i * scale * peakIndexScale)] || 0;
                     let h = Math.round((peak / absmax) * halfH);
 
-                    /* in case of silences, allow the user to specify that we
-                     * always draw *something* (normally a 1px high bar) */
-                    if (h == 0 && this.params.barMinHeight)
-                        h = this.params.barMinHeight;
+                    h = Math.max(h, this.params.barMinHeight);
```

Comment:
So if a bar is smaller of the barMinHeight because the audio peaks in that point reflects a "quieter" audio, that peaks height will be replaced by the barMinHeight parameter? I think it is a bit misleading..

Reply:
Well, otherwise it looks rather awkward as well. I've set it to `4`, and the silent peaks were 4 time higher than the quiet ones. I really think this is the way it has to be.

Reply:
Hi, has this change verified yet? We really need this feature otherwise some parts of signal would be discontinuous.